app/renderer/js/utils/domain-util.js

```diff
@@ -270,7 +272,8 @@ class DomainUtil {
 			ca: (certificateLocation) ? certificateLocation : '',
 			proxy: proxyEnabled ? ProxyUtil.getProxy(url) : '',
 			ecdhCurve: 'auto',
-			headers: { 'User-Agent': SystemUtil.getUserAgent() }
+			headers: { 'User-Agent': SystemUtil.getUserAgent() },
+			rejectUnauthorized: (server.ignoreCerts) ? !server.ignoreCerts : true
```

Comment:
Wouldn't it be more readable to just write `rejectUnauthorized: !server.ignoreCerts`? Spent a couple of minutes more than I should have on this. :P

Reply:
Thanks for your review @kanishk98 . That is done because previous organizations already added into domains.json file would not have the attribute ignoreCerts, and assigning rejectUnauthorized an unknown value can cause problems, as the function checkDomain() is called every time the application restarts. Thanks.

Reply:
The PR is still WIP btw.